packages/foreman/rubygem-sass-rails/rubygem-sass-rails.spec

```
@@ -35,6 +28,7 @@ Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
 # end specfile generated dependencies
 
 Obsoletes: tfm-ror52-rubygem-%{gem_name} <= 5.0.7
+Obsoletes: %{scl_prefix}rubygem-tilt < 2.0.8-4
```

Linter doesn't like this, try 
```suggestion
Obsoletes: tfm-rubygem-tilt < 2.0.8-4
```

---

The question mark did the trick :)